scripts/partition_manager_report.py

```diff
@@ -62,8 +62,8 @@ def parse_args():
     parser = argparse.ArgumentParser(
         description='Parse given Partition Manager output YAML file and print a pretty report',
         formatter_class=argparse.RawDescriptionHelpFormatter)
-    parser.add_argument("-i", "--input", required=True, type=str,
-                        help="Path to the YAML file from Partition Manager")
+    parser.add_argument("-i", "--input", required=True, type=str, nargs="*",
```

Comment:
I see this is using `*` which might actually save you from my earlier comment regarding generator expression.
But i'm unsure if the `*` is intended, cause in my opinion this can lead to hard to find errors.

As example, if an error in CMake code mistakenly creates:
```
partition_manager_report.py  --input
```
when there should have been a list of files, then there will be no error message

However, if the `*` is changed to `+`, then the following will be printed:
```
partition_manager_report.py: error: argument -i/--input: expected at least one argument
```
making it easier to discover errors in other parts of the system.

Reply:
Its intended due to the hack mentioned in the comment above. If the `partition_manager` target does not exist, the `--quiet` argument will be sent to the script, and the user will not see any output from this script.